recipe/meta.yaml

```diff
@@ -11,30 +15,36 @@ source:
     sha256: 69e81178839594049d84526b8f9612825f238f7cfe1f6d579bd6461ed48e5aa0
 
 build:
-  number: 0
+  number: {{ build_number }}
   skip: true  # [win or osx]
+  features:
+    - blas_{{ variant }}
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
+    - pkgconfig
   host:
     - cython >=0.24
-    - pip
+    - setuptools
     - boost
     - python
     - numpy
     - pandas  >=0.15.0
-    - armadillo >=6.500.0
+    - armadillo >=6.500.0,<8.300.0
     - pytest-runner
+    - blas 1.1 {{ variant }}
+    - openblas
   run:
     - boost
     - python
-    - numpy
+    - {{ pin_compatible('numpy') }}
     - pandas  >=0.15.0
-    - armadillo >=6.500.0
-
+    - armadillo >=6.500.0,<8.300.0
```

Comment:
Could you please open an issue at `conda-forge-pinning` about how to pin `armadillo`?

Reply:
Sure! I'm going to open an issue there, thanks! :)